### PR TITLE
updated deploy.sh for deployment docker on UAT

### PR DESCRIPTION
**Pull request by weiweishi**
related documentation is at https://github.com/ualbertalib/di_internal/blob/uat-env/System-Adminstration/UAT-Environment.md

### CODE REVIEW

File: predeploy.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# This script will be sitting in /home/deploy directory, and is for setting up directories and pulling required files for docker deployment
+# This script should be either rsynced, or manually copied to the UAT server when setting up fresh server. 
+BRANCH='master'
+DIR=/home/deploy/jupiter
+mkdir -p $DIR
+mkdir -p $DIR/config
+#curl -o $DIR/deploy.sh https://raw.githubusercontent.com/ualbertalib/jupiter/$BRANCH/deploy.sh
```
Comment (murny) [inline predeploy.sh]: Should this be commented out?

File: predeploy.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+curl -o $DIR/docker-compose.deployment.yml https://raw.githubusercontent.com/ualbertalib/jupiter/$BRANCH/docker-compose.deployment.yml
+curl -o $DIR/config/nginx.conf https://raw.githubusercontent.com/ualbertalib/jupiter/$BRANCH/config/nginx.conf
+if [[ ! -e .env_deployment ]]; then
+curl -o $DIR/.env_deployment https://raw.githubusercontent.com/ualbertalib/jupiter/$BRANCH/.env_deployment_sample
+fi
+source $DIR/deploy.sh
```
Comment (murny) [inline predeploy.sh]: especially if your calling it down here?

Comment (weiweishi) [reply predeploy.sh]: yes, sorry for the oversight. updated.

File: predeploy.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# This script will be sitting in /home/deploy directory, and is for setting up directories and pulling required files for docker deployment
+# This script should be either rsynced, or manually copied to the UAT server when setting up fresh server.
```
Comment (murny) [inline predeploy.sh]: Could also link to the DI docs here once that gets merged too

**Pull request merged**